packages/components/src/common/Notifications/NotificationIcon/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./NotificationIcon";
```

Comment:
this is pretty nitty, but I think it'd be fine if all the `NotificationIcon` files were directly under `Notifications/` instead of in another subfolder. I realize you probably used `plop` which did this automatically

Reply:
nvm!! just saw your DismissButton PR so this makes more sense